build/ci_utils/src/cache/goodie/graalvm.rs

```diff
@@ -166,14 +179,55 @@ mod tests {
     /// Check that we correctly recognize both the GraalVM version and the Java version.
     #[test]
     fn version_recognize() {
-        let version_string = r"openjdk version 17.0.6 2023-01-17
-OpenJDK Runtime Environment GraalVM CE 22.3.1 (build 17.0.6+10-jvmci-22.3-b13)
-OpenJDK 64-Bit Server VM GraalVM CE 22.3.1 (build 17.0.6+10-jvmci-22.3-b13, mixed mode, sharing)";
+        let version_string = "openjdk version \"17.0.7\" 2023-04-18
```

Comment:
Why remove raw-string literal? Now I cannot just copy-paste output, as the quotes need to be escaped.

Reply:
I was not aware how to do raw string literal with double quotes inside. But now I have checked that out and fixed that in 7389e4191986dbc49916ab9a64af46bee300c1a0